molecule/kafka-connect-replicator-plain-kerberos-rhel/molecule.yml

```diff
@@ -176,6 +174,8 @@ provisioner:
         ssl_signed_cert_filepath: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/generated_ssl_files/{{inventory_hostname}}-ca1-signed.crt"
         ssl_key_filepath: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/generated_ssl_files/{{inventory_hostname}}-key.pem"
         ssl_key_password: keypass
+        zookeeper_kerberos_principal: "zookeeper/{{inventory_hostname}}.confluent@{{kerberos.realm | upper}}"
```

Comment:
Here (and at couple of other similar places) also, do we need env var?

Reply:
We are keeping zookeeper for test MDS clusters.